features/support/world.ts

```diff
@@ -60,25 +60,30 @@ export class World {
       })
     } else {
       const stdout = new PassThrough()
+      const stderr = new PassThrough()
```

Comment:
Are these changes still needed? Appears its simpler to not have them

Reply:
True, removed now